PianoFromAbove/MainProcs.cpp

```diff
@@ -1110,7 +1103,7 @@ INT_PTR LoadingProc(HWND hwnd, UINT msg, WPARAM wparam, LPARAM lparam) {
     }
     case WM_TIMER: {
         // lots of race conditions possible but hopefully nobody notices them
-        const char* desc = "placeholder";
+        const char* desc = "Loading...";
```

Comment:
This was intentional. This message wouldn't show up unless something went *very* wrong with the MIDI loader.